ods_ci/tests/Resources/CLI/Minio.resource

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+Documentation       Collcetion of keywords to deploy and remove MinIO storage
+Library             OperatingSystem
+Resource            ../Common.robot
+
+
+*** Variables ***
+${MINIO_RESOURCES_DIRPATH}=    ods_ci/tests/Resources/Files/minio
+${MINIO_POD_FILEPATH}=    ${MINIO_RESOURCES_DIRPATH}/minio.yaml
+${MINIO_SECRET_FILEPATH}=    ${MINIO_RESOURCES_DIRPATH}/minio_secret.yaml
+${MINIO_SA_FILEPATH}=    ${MINIO_RESOURCES_DIRPATH}/minio_serviceaccount.yaml
+
+
+*** Keywords ***
+Deploy MinIO
+    [Documentation]    Deploys MiniIO pod, service and route using the given ${minio_image}
+    [Arguments]    ${minio_image}=quay.io/modh/ods-ci-minio-models:1.0
```

Comment:
Use digest for the compatibility for disconnected cluster

Reply:
ok

Reply:
Hi @bdattoma why do you deploy the Minio? In a disconencted env it is already available.
It is enough just to check test-variables.yaml

Reply:
will discuss offline